### PR TITLE
WIP: Add bower badge

### PR DESCRIPTION
**Pull request by ahmadkarlam**
Adding bower badge @sagarpreet-chadha.

I add 2 different badges. The first badge quite nicer than the second one. But the first badge, I can't find leaflet-environmental-layers on [Bower](https://bower.io/search). 

Screen Shot 2019-05-21 at 10 45 31

### CODE REVIEW

**Comment by ahmadkarlam** [comment]
That sounds good.
Sorry, I forget to delete bower link below.

Which one you wanna use as a badge? Left or right?

**Comment by sagarpreet-chadha** [comment]
No problem 👍 😄 ! I think let's keep both of them :)
Thanks !!!

**Comment by sagarpreet-chadha** [comment]
Hey @sashadev-sky , Yes LEL is registered on bower :  


Bower is still used by many people , infact currently Mapknitter uses bower . I think if we cannot get a clickable badge then we should atleast tell users that we are present on bower also . What do you think ? Thank you :balloon: :smile:

**Comment by sashadev-sky** [comment]
@sagarpreet-chadha Oh I see, you can't search half of a word in bower. 

Ok so what if we just change the link of the first badge (the one that is more aesthetic, to take the user to https://bower.io/search/ like the second badge currently does) - and then delete the 2nd badge?

**Comment by sagarpreet-chadha** [comment]
Sounds awesome !!!

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Leaflet Environmental Layers
 
 [![npm version](https://badge.fury.io/js/leaflet-environmental-layers.svg)](https://badge.fury.io/js/leaflet-environmental-layers)
+[![Bower version](https://badge.fury.io/bo/bower.svg)](https://badge.fury.io/bo/bower)
```
Comment (sashadev-sky) [inline README.md]:
```suggestion
```

Comment (sashadev-sky) [inline README.md]:
```suggestion

```

**Comment by ahmadkarlam** [comment]
> @ahmadkarlam I made a commit suggestion to delete the first badge and just keep the 2nd as per conversation with @sagarpreet-chadha.
> 
> Then we just want to delete the part in the README about bower:
> 
> ```
> ##### Bower :
> Search leaflet-environmental-layers here : https://bower.io/search/
> ```
> 
> and this is ready for merge! Thank you for your help 😊🙏

Thanks for your suggestion
I already delete bower link and merge the suggestion

**Comment by sashadev-sky** [comment]
@ahmadkarlam great thank you! merged 🎉

**Pull request merged**